frontend/app/src/components/history/transactions/events/TransactionEventForm.vue

```diff
@@ -27,26 +26,23 @@ import { convertFromTimestamp, convertToTimestamp } from '@/utils/date';
 import { useEventTypeData } from '@/utils/history';
 import { toMessages } from '@/utils/validation-errors';
 
-const props = defineProps({
-  value: { required: false, type: Boolean, default: false },
-  edit: {
-    required: false,
-    type: Object as PropType<EthTransactionEvent | null>,
-    default: null
-  },
-  transaction: {
-    required: false,
-    type: Object as PropType<EthTransactionEntry | null>,
-    default: null
+const props = withDefaults(
+  defineProps<{
+    value?: boolean;
+    edit?: EthTransactionEvent | null;
+    transaction?: EthTransactionEntry | null;
+  }>(),
+  {
+    value: false,
+    edit: () => null,
+    transaction: () => null
```

Comment:
Why `() => null` instead of `null`?

For arrays and objects, we use it because otherwise, they will pass a reference. But null is a special case and we can treat it like a primitive.

Reply:
I remember it threw an error in the past, but I don't see it anymore 😅, will change it